README.md

```diff
@@ -287,6 +291,45 @@ This method provides a list of available locales configured in your FirstSpirit
   Promise<string[]>
 ```
 
+### Fetch From CaaS
+
+#### Using `remoteApi.buildCaaSUrl()` and Authorization Headers to Send a Request to CaaS
+
+The Content as a Service (CaaS) provides a powerful way to retrieve and manage content. This guide focuses on how to construct and send requests to the CaaS service using the `buildCaasUrl` function and the aggregation URI `/_aggrs/example`. Additionally, it covers the necessary steps to include authorization headers for secure communication.
+
+In this tutorial, we will explore how to use the JavaScript Content API Library's `remoteApi.buildCaaSUrl()` method to construct a CaaS URL and how to include authorization headers for sending a fetch request to the CaaS platform.
+
+#### Prerequisites
```

Comment:
in the specific "extends api" example of the "fetch aggregation" -  we would also require an aggregation name - (the url to the aggr can be derived by the name of it)